### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by bnmajor**
Addresses #16

### CODE REVIEW

**Comment by bnmajor** [comment]
@thewtex This covers most of the topics covered in #16 but there are some pieces that are missing. I need some clarification on what the following sections should be actually be covering:

- Deployments
  - Server app
  - Desktop app
- Integrations: Should this just be covering installation and maybe an example of using that dependency?
- Jupyterlite and VSCode are not yet covered in the docs as there are still issues with both that I am trying to sort out. VSCode is not displaying the viewer (although it also does not produce errors and works perfectly in Jupyter) and Jupyterlite has issues with installation as it requires pure python wheels for all packages (must be installed with `piplite`):
  ```
  ValueError: Couldn't find a pure Python 3 wheel for: 'multidict<5.0,>=4.5', 'yarl<2.0,>=1.0'
  ```

**Review by thewtex — APPROVED** [review]

**Comment by thewtex** [comment]
>  I need some clarification on what the following sections should be actually be covering

Let's cover the sections we do not have content for in a follow-up PR.

> Integrations: Should this just be covering installation and maybe an example of using that dependency?

:+1: both :-)

File: docs/deployments.rst [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+To use itkwidgets locally first `install Jupyter Notebook`_ and start Jupyter:
+
+.. _install Jupyter Notebook: https://jupyter.org/install#jupyter-notebook
+
+.. code-block::
+
+    pip install notebook
+    jupyter notebook
+
+If you'd rather interact with reomtely hosted notebooks you can also open them
```
Comment (labdala) [inline docs/deployments.rst:39]: small detail "remotely"

**Review by labdala — APPROVED** [review]

**Review by thewtex — APPROVED** [review]
Loving the screeshots! :heart_eyes:

**Comment by bnmajor** [comment]
@thewtex This adds the integrations section as well as screenshots, support for the listed integrations, and new notebook examples. Two things to address though:
1. I do not have permissions to add a webhook to get `readthedocs` set up
2. Was the zarr example notebook that you added working for you? I was able to download the data locally and test it to get the screenshot but actually running the notebook does not work for me (I can open a separate issue/PR for this if this is a bug)

File: docs/deployments.rst (hunk 1 above)
Comment (bnmajor) [reply docs/deployments.rst]: Thanks for catching this!

**Comment by thewtex** [comment]
> I do not have permissions to add a webhook to get readthedocs set up

Could you please add my account to the ReadTheDocs project?

> Was the zarr example notebook that you added working for you? I was able to download the data locally and test it to get the screenshot but actually running the notebook does not work for me (I can open a separate issue/PR for this if this is a bug)

Yes, it was -- please create an issue.

Thanks!

**Comment by bnmajor** [comment]
> > I do not have permissions to add a webhook to get readthedocs set up
> 
> Could you please add my account to the ReadTheDocs project?

I added you as a maintainer so you should be able to access it now: https://readthedocs.org/projects/itkwidgets/. If not let me know, there may be additional steps I need to take.

> > Was the zarr example notebook that you added working for you? I was able to download the data locally and test it to get the screenshot but actually running the notebook does not work for me (I can open a separate issue/PR for this if this is a bug)
> 
> Yes, it was -- please create an issue.

Great will do, thanks!

**Comment by bnmajor** [comment]
> Thanks. I enabled the GitHub Integration. It should be working now.
Thanks! I think everything is all set with that now, docs should build on pull requests and the default branch is set to main.

**Pull request merged**